README.md

```diff
@@ -16,55 +16,57 @@ First, you'll need a `vercel.json` file in your project:
 {
   "functions": {
     "api/**/*.rs": {
-      "runtime": "vercel-rust@3.1.2"
+      "runtime": "vercel-rust@4.0.0"
     }
   }
 }
 ```
 
-A Vercel Function will be created for every file that matches `api/**/*.rs`. Next, you can create a new Function `api/user.rs`:
+A Vercel Function will be created for every file that matches `api/**/*.rs`.
+
+Example:
 
 ```rust
-use http::{StatusCode};
-use vercel_lambda::{lambda, error::VercelError, IntoResponse, Request, Response};
-use std::error::Error;
-
-fn handler(_: Request) -> Result<impl IntoResponse, VercelError> {
-	let response = Response::builder()
-		.status(StatusCode::OK)
-		.header("Content-Type", "text/plain")
-		.body("Hello World")
-		.expect("Internal Server Error");
-
-		Ok(response)
+use serde_json::json;
+use vercel_runtime::{
+    lambda_http::{http::StatusCode, Error as LambdaError, Response},
+    run, IntoResponse, ProxyError, ProxyRequest,
+};
+
+#[tokio::main]
+async fn main() -> Result<(), LambdaError> {
+    run(handler).await?;
+    Ok(())
 }
 
-// Start the runtime with the handler
-fn main() -> Result<(), Box<dyn Error>> {
-	Ok(lambda!(handler))
+pub async fn handler(_req: ProxyRequest) -> Result<impl IntoResponse, ProxyError> {
```

Comment:
See our chat about `Proxy` being confusing naming. Think we can remove it and just have `Request / Response / Error`